File: problems/problem_6/fullboilerplate.java
import java.util.*;
public class Main {
    // {{USER_CODE}}
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.hasNext() ? sc.next() : "";
        String t = sc.hasNext() ? sc.next() : "";
        boolean res = isAnagram(s, t);
        System.out.println(res ? "true" : "false");
    }
}

File: problems/problem_6/fullboilerplate.cpp
#include <bits/stdc++.h>
using namespace std;
// {{USER_CODE}}
int main() {
    string s, t;
    if (cin >> s >> t) {
        bool res = isAnagram(s, t);
        cout << (res ? "true" : "false") << endl;
    }
    return 0;
}